README.md

```diff
@@ -1,6 +1,6 @@
 ## Script
 ```lua
-local owner = "Upbolt"
+local owner = "DzGak"
```

Comment:
don't know if this is something you're still actively working on, but if it isn't then this part is gonna have to change back